forest/cli/src/cli/send_cmd.rs

```diff
@@ -7,9 +7,98 @@ use forest_json::message::json::MessageJson;
 use forest_rpc_client::{mpool_push_message, wallet_default_address};
 use fvm_shared::{address::Address, econ::TokenAmount, message::Message, METHOD_SEND};
 use num::BigInt;
+use rust_decimal::prelude::*;
+use rust_decimal_macros::dec;
 
 use super::{handle_rpc_err, Config};
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct FILAmount {
+    pub value: TokenAmount,
+}
+
+impl FromStr for FILAmount {
+    type Err = anyhow::Error;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        let suffix_idx = s.rfind(char::is_numeric);
+        let (val, suffix) = match suffix_idx {
+            Some(idx) => s.split_at(idx + 1),
+            None => return Err(anyhow::anyhow!("failed to parse string: {}", s)),
+        };
+        let suffix = suffix.replace(' ', "");
```

Comment:
Does this mean that, say, `fem to fil` is valid?

Reply:
It did erroneously allow an amount with units `fem to fil`, but I have corrected that.